### DISCUSSION TITLE
Ask Consent Form(Checkboxes) for Recording only first-time for logged in users?

### DISCUSSION THREAD

**Discussion by pritamnanda** [THM-Health/PILOS#1445] [General]
Hello everyone,

In the current release of V4, there is a consent form related to recordings & logging every time the Room owner or the Moderator Starts the Room.

I agree that it's necessary for the Attendees or the Guests, but can we implement account-based preferences or consent for registered & logged-in users, which will allow users to save their choices & use that for room starts every time with one-click?

Maybe we can add an option if the consent should be taken every time or just use the saved one.

I couldn't find any such related settings or preferences. @SamuelWei Is there any reason for not implementing this or it can be considered as a feature request ?

Thanks for your attention & time.

![image](https://github.com/user-attachments/assets/dcff6d6c-66aa-4cf4-bc97-58c8fcdadf4b)

**Comment by samuelwei** [comment]
Moved to issue #1446
